gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -138,6 +140,12 @@ gst_tensor_query_client_class_init (GstTensorQueryClientClass * klass)
           "The main topic of the host.",
           "", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_TIMEOUT,
+      g_param_spec_uint ("timeout", "timeout value",
+          "A timeout value (in ms) to wait message from server", 0,
```

Comment:
What's the behavior if timeout == 0?

Reply:
It there is no element in queue, do not wait and do not call `gst_pad_push`

Reply:
Describe the behaviors for tensor_query_client users. What should they expect when they set timeout? Is the behavior kept the same as before if it's set 0? Usually, users will expect that if it's 0, it will wait for the server forever.

Reply:
Chill, users won't see the word unless they look into code.
With default 0 timeout, it behaves same as before - It does not wait for the message from tensor_query_serversink to be queued.
@jaeyun-jung and I discussed about this, and we chose default 0 timeout to "no wait" and do not allow infinite wait.

Reply:
What I'm telling you is that you need to describe it in ```g_param_spec_unit```. This is not an internal string for code readers. GStreamer users see these with gst-inspect, which is the default utility for gstreamer users. We have users who do not use ml-api, but use gstreamer api.

Reply:
Updated the description as
"A timeout value (in ms) to wait message from query server after sending buffer to server. 0 means no wait."
PTAL